onnxruntime/core/providers/cuda/reduction/reduction_ops.cc

```diff
@@ -65,7 +64,11 @@ namespace cuda {
       T,                                                                                   \
       kCudaExecutionProvider,                                                              \
       (*KernelDefBuilder::Create()).TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
-      name<T>);                                                                            \
+      name<T>);
+
+// Register those with changes in OpSet12.
+#define REGISTER_KERNEL_TYPED_12(name, T)                                                  \
+  REGISTER_KERNEL_VERSIONED_TYPED_12(name, T)                                              \
```

Comment:
why is macro named 12 but has version 13 inside?

Reply:
good catch! I just keep the initial name. Will change it to 13